affiliate/blocks/productCard/src/Save.jsx

```diff
@@ -1,14 +1,13 @@
 import { useBlockProps } from '@wordpress/block-editor';
 
-import ProductCarousel from './ProductCard';
```

Comment:
Well, this certainly won't do. I modified this file and forgot to push the change in my previous PR.

Reply:
I've been testing on a feature branch, so nothing is broken in production.